Apeswap-Bots/Gnana-fee-update/src/agent.spec.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  Finding,
+  FindingType,
+  FindingSeverity,
+  HandleTransaction,
+  ethers,
+} from "forta-agent";
+import {
+  createAddress,
+  TestTransactionEvent,
+} from "forta-agent-tools/lib/tests";
+import { handleTransaction } from "./agent";
+import utils from "./utils";
+
+const WRONG_EVENT_ABI: string[] = [
+  "event Transfer(address indexed from,address indexed to,uint256 value)",
+];
+
+const WRONG_EVENTS_IFACE: ethers.utils.Interface = new ethers.utils.Interface(
+  WRONG_EVENT_ABI
+);
+
+const TEST_GNANA_TOKEN: string = createAddress("0xcdcd");
+
+const previousFee = "1";
+const currentFee = "2";
+
+const findingTestCases = [
+  Finding.fromObject({
+    name: "Detect Fees Related To The Token",
+    description: "Token tax fee has been changed",
+    alertId: "APESWAP-3",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    protocol: "Apeswap",
+    metadata: {
+      previousFee: previousFee,
+      currentFee: currentFee,
+    },
+    addresses: [],
+  }),
+  Finding.fromObject({
```

Comment:
You can avoid doing this by making it a Finding function (similar to the one you use in the agent logic) and passing the Fees in there